src/scf/mixing.jl

```diff
@@ -77,11 +77,86 @@ function mix(mixing::RestaMixing, basis, ρin::RealFourierArray, ρout::RealFour
 
     ρin = ρin.fourier
     ρout = ρout.fourier
-    C = 1 - εr
+    C0 = 1 - εr
     Gsq = [sum(abs2, basis.model.recip_lattice * G) for G in G_vectors(basis)]
 
-    ρnext = @. ρin + T(mixing.α) * (ρout - ρin) * (kF^2 - C * Gsq) / (εr * kF^2 - C * Gsq)
+    ρnext = @. ρin + T(mixing.α) * (ρout - ρin) * (kF^2 - C0 * Gsq) / (εr * kF^2 - C0 * Gsq)
     # take the correct DC component from ρout; otherwise the DC component never gets updated
     ρnext[1] = ρout[1]
     from_fourier(basis, ρnext)
 end
+
+
+@doc raw"""
+The model for the susceptibility is
+```math
+\chi_0(r, r') = (-LDOS(εF, r) + LDOS(εF, r) LDOS(εF, r') / DOS(εF))
+              + \sqrt{L(x)} IFFT \frac{C_0 G^2}{4π (1 - C_0 G^2 / k_F^2)} FFT \sqrt{L(x)}
+```
+where ``C_0 = 1 - ε_r`` and the same convention for parameters is used as in `RestaMixing`.
+Additionally there is the real-space localisation function `L(r)`.
+"""
+struct HybridMixing
+    α::Real
+    εr::Real
+    kF::Real
+    localisation::Function  # Function L(r), `r` is in fractional real-space coordinates
```

Comment:
Zation

Reply:
Maybe this is a good place for using this fancy macro to make structs have defaults